src/types.ts

```diff
@@ -64,14 +71,14 @@ function entryFromObject(message) {
     return entry;
 }
 
-function entryFromString(message) {
+function entryFromString(message: Message): any {
     return {
         Id: message,
         MessageBody: message
     };
 }
 
-function entryFromMessage(message) {
+export function entryFromMessage(message: any): string[] {
```

Comment:
The appropriate aws entry type can be returned from this function; should return the same type as `entryFromString` and `entryFromObject`, refactored.